### PR TITLE
Mobile middle section

### PR DESCRIPTION
**Pull request by Bleu-Yves-Sopoude**
In this new feature we :

-Added Work Section
-Added About Section
-Added Working Html & CSS linters
-Used ul and li for categories in "Work" section
-Implemented the button interactions with Hover
-Used responsive values in CSS rules

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,47 +9,154 @@
...
+          <div class="description">
+            <h2>Project name goes here</h2>
+            <ul>
+              <li>HTML/CSS</li>
+              <li>Ruby on Rails</li>
+              <li>JavaScript</li>
+            </ul>
+            <button type="button">
+              See this project
+              <img src="static/Union.png" alt="right arrow icon" />
```
Comment (bobb-Rob) [inline index.html:25]: - [ ] This icon should align to the center in the cross-axis and should NOT have a border around it. Kindly fix this.

Your work
image

Figma Design
image

File: index.html [hunk 2]
```diff
@@ -9,47 +9,154 @@
...
-        </section>
-
-
-    </main>
-
-	
-	
-</body>
+	<section class="work">
+        <h2 class="projects">Projects</h2>
```
Comment (bobb-Rob) [inline index.html:13]: - [ ] The font-style of this section title should be `Playfair Display`, but currently, it is not. I see that have applied it in your CSS file, but it still does not affect what we see on the screen. Kindly fix this

  Your work
  image

   Figma 
  image

File: index.html [hunk 3]
```diff
@@ -9,47 +9,154 @@
...
+              See this project
+              <img src="static/Union.png" alt="right arrow icon" />
+            </button>
+          </div>
+        </div>
+      </section>
+  
+      <section class="about">
+        <div class="about-description">
+          <h2>About</h2>
```
Comment (bobb-Rob) [inline index.html]: - [ ] Similarly, the `font-family` here is NOT `Playfair display` as we can see on the screen, even though you have used it in your CSS. Kindly fix this.

Figma design
image

Your work
image

File: index.html [hunk 4]
```diff
@@ -9,47 +9,154 @@
...
+            <li>Skills</li>
+            <li>Database Management</li>
+            <li>CLI</li>
+            <li>API Design</li>
+            <li>Version Control</li>
+            <li>Web Development</li>
+          </ul>
+          <div class="idea"></div>
+        </div>
+      </section>
```
Comment (bobb-Rob) [inline index.html]: - [ ] Thi about section should have a left and right padding as seen in the Figma design. Kindly ensure you implement this.

Your work
image

Figma Design
image

You can see in the Figma design, that there is a white space on the left and right sides, and the `Get my Cv` button spans the whole width of the screen.

File: index.html [hunk 5]
```diff
@@ -9,47 +9,154 @@
...
+          <ul class="skills">
+            <li>Skills</li>
+            <li>Database Management</li>
+            <li>CLI</li>
+            <li>API Design</li>
+            <li>Version Control</li>
+            <li>Web Development</li>
+          </ul>
+          <div class="idea"></div>
+        </div>
```
Comment (bobb-Rob) [inline index.html]: - [ ] The technology categories ("Languages", "Frameworks" and "Skills") should be in a `ul`, and each of them in a `li`. Also, the technology in each category(e.g., "JavasScript", "HTML" and "CSS") is also a list `ul` and each of them a `li`. It is basically a nested `ul` with its own `li` in each of the `li` of the outer `ul`. Kindly fix this as this improve the accessibility of the page mostly for blind users who depends on using the keyboard `tab` key to navigate the webpage.

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @Bleu-Yves-Sopoude, 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 6]
```diff
@@ -9,47 +9,157 @@
...
+      </ul>
```
Comment (aamirkhan2478) [inline index.html]: - [ ] As mentioned by the previous reviewer :point_up: the technology categories ("Languages", "Frameworks" and "Skills") should be in a `ul`, and each of them in a `li`. Also, the technology in each category(e.g., "JavasScript", "HTML" and "CSS") is also a list `ul` and each of them a `li`. It is basically a nested `ul` with its own `li` in each of the `li` of the outer `ul`. Kindly fix this as this improve the accessibility of the page mostly for blind users who depends on using the keyboard `tab` key to navigate the webpage but still, it's the same you did change it kindly fix it and check below :point_down: example of nested lists :slightly_smiling_face: 

```html
<ul>
     <li>Languages
            <ul>
                 <li>Javascript</li>
                <li>Ruby</li>
                <li>Html</li>
               <li>CSS</li>
            </ul>
     </li>
 <li>Frameworks
            <ul>
                 <li>React.js</li>
                <li>Ruby on rails</li>
                <li>RSpec</li>
               <li>Capibara</li>
            </ul>
     </li>
 <li>Skills
            <ul>
                 <li>Database Management</li>
                <li>CLI</li>
                <li>API Design</li>
               <li>Version Control</li>
               <li>Web Development</li>
            </ul>
     </li>
</ul>
```

File: index.html [hunk 7]
```diff
@@ -9,47 +9,157 @@
...
+            I can help you build a product , feature or <br />
+            website Look through some of my work <br />
+            and experience! If you like what you see <br />
+            and have a project you need coded, <br />
+            don’t hesitate to contact me.
+          </p>
+          <button type="button">
+            Get my CV <img src="static/ic_download.svg" />
+          </button>
+        </div>
```
Comment (aamirkhan2478) [inline index.html]: - [ ] As mentioned by the previous reviewer :point_up: this about section should have a left and right padding as seen in the Figma design. Kindly ensure you implement this. well done :clap: you did changes given by the previous reviewer but still, there are some issues there kindly check below :point_down: the comparison of your and Figma's design :slightly_smiling_face: 

**Your Design**
Screenshot from 2023-05-05 23-50-29

**Figma Design**
236502126-ea3bf922-9a1b-4be2-98c1-53bcfc38437e

- [ ] You can see in the Figma design, that there is a white space on the left and right sides, and the Get my Cv button spans the whole width of the screen.

**Review by aamirkhan2478 — CHANGES_REQUESTED** [review]
Hi @Bleu-Yves-Sopoude , 



Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

 ## Highlights
✔️ Good Effort
✔️ Your design is good.
✔️ All linters passed

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 6 above)
Comment (Bleu-Yves-Sopoude) [reply index.html]: templates

Hi @aamirkhan2478 and @bobb-Rob ,
When I add the tags you tell me, there are dots that appear near the titles which is not on the design. Now the instruction is to stay true to the design.

**Review by braincee — APPROVED** [review]
Hi @Bleu-Yves-Sopoude 👋  , I am @braincee, your Code Reviewer.

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

youre-approved-shirley-auto-group

OPTIONAL CHANGES
---

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @braincee._

---

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**